src/components/PayWithCard.tsx

```diff
@@ -29,12 +30,20 @@ interface PayWithCardProps {
   onError?: (error: PaperSDKError) => void;
 }
 
+type SignedPayload = {
```

Comment:
Can we just do a recursive `type CustomArgType = Record<string, string|Boolean|number|CustomArgType>`? A more general type is going to invalidate `SignedPayloadType` anyways

Reply:
The main thing is that I want to give them typing. I think if we expand the prop to be more than 1 type we should add `SignedPayloadType | ...`. I think types should be as narrow as possible instead of general. What do you think?